### PR TITLE
updated integration test for Teller and Customer

### PR DESCRIPTION
**Pull request by jfiricel**

### CODE REVIEW

File: Prospect-iris/src/test/java/com/temenos/marketplace/TestTeller_LcyCashinCashwdlITCase.java [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+/**
+ * TODO: Document me!
+ *
+ * @author jfiricel
+ *
+ */
+public class TestTeller_LcyCashinCashwdlITCase {
+
+    //the user below is a teller one that should be present into the testing area, otherwise it should be changed with another one
+    public static String TELLER_USER_NAME = "CSAGENT";
```
Comment (mjangid) [inline Prospect-iris/src/test/java/com/temenos/marketplace/TestTeller_LcyCashinCashwdlITCase.java]: Hi @jfiricel 
Can you please remove this TELLER_USER_NAME and get it from Configuration file

**Review by mjangid — COMMENTED** [review]
Please get the user name from configuration file

**Comment by jfiricel** [comment]
Please merge

**Comment by jfiricel** [comment]
Please merge also tests and latest enquiries

**Comment by jfiricel** [comment]
Please merge

**Comment by jfiricel** [comment]
Please merge also the addition of enqStmtEntLasts and enqStmtEntBooks APIs

**Pull request merged**